src/handlers/assign.rs

```diff
@@ -804,14 +809,18 @@ async fn find_reviewer_from_names(
 
     if filtered_candidates.is_empty() {
         // NOTE: disabled for now, just log
-        log::info!("Filtered list of PR assignee is empty");
+        log::info!("[#{}] Filtered list of PR assignee is empty", issue.number);
         // return Err(FindReviewerError::AllReviewersFiltered {
         //     initial: names.to_vec(),
         //     filtered: names.to_vec(),
         // });
     }
 
-    log::info!("Filtered list of candidates: {:?}", filtered_candidates);
+    log::info!(
```

Comment:
We should really migrate to tracing and use spans to get have proper log context :)